packages/core/prime.css

```diff
@@ -15,3 +15,16 @@ button {
 :host([hidden]) {
   display: none;
 }
+
+/* Just give me the CSS classes option */
+.button-link {
```

Comment:
it doesn't look like anything here is related to `link`, should we just call this `.button`? My assumption when looking at this name is that it's a special collection for link buttons, when it's just the primary button styles.